tests/test_file_parser.py

```
@@ -1,493 +1,113 @@
+    ]
+    
+    test_file = None
+    for path in test_files:
+        if os.path.exists(path):
+            test_file = path
+            break
+    
+    if not test_file:
+        pytest.fail(f"No test audio file found. Create one of: {test_files}")
+    
+    result = await parser._parse_audio(test_file)
```

Pylint (3.3.7)

[warning] Access to a protected member _parse_audio of a client class

(W0212)